packages/lib/src/exposes.ts

```diff
@@ -27,7 +27,7 @@ export function exposesPlugin(
   let moduleMap = ''
   const replaceMap = new Map()
   const provideExposes = parseOptions(
-    options.exposes,
+    options.exposes ? options.exposes : {},
```

Comment:
```suggestion
    options.exposes,
```
parseOptions()会处理null的情况